.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - checkout
 
       - restore_cache:
-          key: v1-gems-{{ checksum "Gemfile" }}
+          key: fastlane-macos-gems-{{ checksum "Gemfile" }}
```

Comment:
`v1-macos-gems-{{ checksum "Gemfile" }}` would be a better cache key here, because we need to be able to invalidate the cache key easily

Reply:
Found it: https://circleci.com/docs/2.0/caching/#clearing-cache